agent/kubernetes/service.go

```diff
@@ -119,10 +137,16 @@ func (cs *CostService) storePodCosts() {
 		tags["node_name"] = podInfo.nodeName
 		tags["namespace"] = podInfo.namespace
 
-		podCost, _ := podInfo.cost.Float64()
+		totalCost, _ := podInfo.cost.totalCost.Float64()
+		utilization, _ := podInfo.cost.utilization.Float64()
+		underUtilization, _ := podInfo.cost.underUtilization.Float64()
+		nodOverhead, _ := podInfo.cost.nodeOverhead.Float64()
 
 		fields := map[string]interface{}{
-			"monthly_cost": podCost,
+			"total_cost":        totalCost,
```

Comment:
Same here, can we just go with` total` maybe?

Reply:
Sure!